assertTrue fails when used with pytest tp fixture

I am using pytest and trying to use the `tp` fixture, but I am running into this error on certain methods: `AttributeError: 'TestCase' object has no attribute 'assertTrue'`.  

Here is my trace:
```python
    def test_thing(tp):
        response = tp.get('test-url')
>       tp.assertInContext("foo")

test-project/test-app/tests/my_test.py:11:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <test_plus.plugin.TestCase object at 0x107a9ab80>, key = 'requests'

    def assertInContext(self, key):
        if self.last_response is not None:
>           self.assertTrue(key in self.last_response.context)
E           AttributeError: 'TestCase' object has no attribute 'assertTrue'

.venv/lib/python3.8/site-packages/test_plus/test.py:324: AttributeError
=========================================================================== short test summary info ===========================================================================
FAILED test-project/test-app/tests/my_test.py::test_thing - AttributeError: 'TestCase' object has no attribute 'assertTrue'
```

Not every method runs into this issue.  For example `assertLoginRequired` works.

--- a/test_plus/plugin.py
+++ b/test_plus/plugin.py
@@ -1,7 +1,7 @@
 import pytest
 
 from .compat import get_api_client
-from .test import BaseTestCase
+from .test import TestCase as BaseTestCase
 
 
 class TestCase(BaseTestCase):
@@ -13,12 +13,6 @@ class TestCase(BaseTestCase):
     def __init__(self, *args, **kwargs):
         self.last_response = None
         super(TestCase, self).__init__(*args, **kwargs)
-
-    def assertEqual(self, first, second, msg=None):
-        if not msg:
-            msg = "Elements Not Equal"
-
-        assert first == second, msg
 
 
 @pytest.fixture

--- a/test_project/test_app/tests/test_pytest.py
+++ b/test_project/test_app/tests/test_pytest.py
@@ -16,3 +16,9 @@ def test_api(tp_api):
 
 def test_assert_login_required(tp):
     tp.assertLoginRequired("view-needs-login")
+
+
+def test_assert_in_context(tp):
+    response = tp.get('view-context-with')
+    assert 'testvalue' in response.context
+    tp.assertInContext('testvalue')

